Initial docs (#66)
EventStore/EventStore-Client-Dotnet: docs/docs/reading-events/reading-from-a-stream.md

### configureOperationOptions

This argument is generic setting class for all operations that can be set on all operations executed against EventStoreDB. To find out more see [here]().

### userCredentials
The credentials used to read the data can be supplied. Please see [here](authentication/authenticating-with-username-password.html).
 for more information.

## Reading from a revision

As well as providing a `StreamPosition` you can also provide a specific stream revision in the form of a `ulong`

<<< @/samples/reading-events/Program.cs#read-from-stream-position

## Reading backwards

As well as being able to read a stream forwards you can also go backwards. When reading backwards is the the `StreamPosition` will have to be set to the end if you want to read all the events:

<<< @/samples/reading-events/Program.cs#reading-backwards 

:::tip
You can use reading backwards to find the last position in the stream. Just read backwards one event and get the position.
:::

## Checking for stream presence 

> Reading a stream returns a `ReadStreamResult` this contains a property `ReadState`. This property can have the value `StreamNotFound` and `Ok`.

Reviewer: ```suggestion
Reading a stream returns a `ReadStreamResult`, which contains a `ReadState` property. This will have a value of either `StreamNotFound` or `Ok`.
```
